### PR TITLE
Copy changes to plans to ensure consistency across signup and /plans pages

### PR DESCRIPTION
**Pull request by sararosso**
Updated taglines for Blogger, Personal, Premium, Business, and
eCommerce.

Repeated Business features on the eCommerce plan full feature view and
new eComm-specific features at the bottom like other plans.

Updated Blogger domain feature + tooltip.

Added more descriptive tooltips for new eComm features.

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -241,10 +241,10 @@ const getPlanPersonalDetails = () => ( {
 	group: GROUP_WPCOM,
 	type: TYPE_PERSONAL,
 	getTitle: () => i18n.translate( 'Personal' ),
-	getAudience: () => i18n.translate( 'Best for hobbyists' ),
-	getBlogAudience: () => i18n.translate( 'Best for hobbyists' ),
-	getPortfolioAudience: () => i18n.translate( 'Best for hobbyists' ),
-	getStoreAudience: () => i18n.translate( 'Best for hobbyists' ),
+	getAudience: () => i18n.translate( 'Best for personal use' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Best for Personal Use' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-features)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -291,10 +291,10 @@ const getPlanPremiumDetails = () => ( {
 	group: GROUP_WPCOM,
 	type: TYPE_PREMIUM,
 	getTitle: () => i18n.translate( 'Premium' ),
-	getAudience: () => i18n.translate( 'Best for entrepreneurs' ),
-	getBlogAudience: () => i18n.translate( 'Best for professionals' ),
-	getPortfolioAudience: () => i18n.translate( 'Best for freelancers' ),
-	getStoreAudience: () => i18n.translate( 'Best for freelancers' ),
+	getAudience: () => i18n.translate( 'Best for entrepreneurs & freelancers' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Best for Entrepreneurs and Freelancers' )` *ES Score: 23*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-features)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -354,13 +354,13 @@ const getPlanBusinessDetails = () => ( {
...
 	getAudience: () => i18n.translate( 'Best for small businesses' ),
-	getBlogAudience: () => i18n.translate( 'Best for brands' ),
+	getBlogAudience: () => i18n.translate( 'Best for small businesses' ),
 	getPortfolioAudience: () => i18n.translate( 'Best for small businesses' ),
 	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
 	getDescription: abtest => {
 		if ( abtest && abtest( 'businessPlanDescriptionAT' ) === 'pluginsAndThemes' ) {
 			return i18n.translate(
-				'{{strong}}Best for Small Business:{{/strong}} Power your' +
+				'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( '{{strong}}Best for Small Business:{{/strong}} Power your business website with custom plugins and themes, unlimited premium and business theme templates, Google Analytics support, unlimited storage, and the ability to remove WordPress.com branding.' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 4]
```diff
@@ -374,7 +374,7 @@ const getPlanBusinessDetails = () => ( {
 			);
 		}
 		return i18n.translate(
-			'{{strong}}Best for Small Business:{{/strong}} Power your' +
+			'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( '{{strong}}Best for Small Business:{{/strong}} Power your business website with unlimited premium and business theme templates, Google Analytics support, unlimited storage, and the ability to remove WordPress.com branding.' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-features)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by sararosso** [comment]
Testing instructions: 

Go to WordPress.com/plans to see the new tabbed experience and the applied copy changes. 

In addition, go to My Sites > Add new site and on the Plans page of the signup process, you should see updated copy and tooltips for all the features.

**Comment by sararosso** [comment]
Debugging my own work I noticed that the Premium tagline pushes to two lines in /signup. :/

screen shot 2018-11-29 at 8 11 28 am

**Comment by apeatling** [comment]
@sararosso Tested and confirmed seeing the tabs, and all copy looks good. :+1: I assume the new tooltip content will match on the homepage plans display too?

**Comment by jaswrks** [comment]
- Should _"All free features"_ be _"All Free features"_?
- Should this be "All Blogger Features"? I think it works either way, actually.

**Comment by sararosso** [comment]
> @sararosso Tested and confirmed seeing the tabs, and all copy looks good. 👍 I assume the new tooltip content will match on the homepage plans display too?

Yes! @jaswrks and I are working on making sure those are synced - this will be the 'master' for the home page to inherit.

**Comment by sararosso** [comment]
> * Should _"All free features"_ be _"All Free features"_?

Thanks for looking!

Those are actually both correct - the free plan technically isn't capitalized since it's not a plan - I've seen it elsewhere but also found that confirmed in the brand book: 
> The word “free” is not a plan name, and shouldn’t be capitalized
either:

> * Should this be "All Blogger Features"? I think it works either way, actually.
No, as the Personal plan inherits the free features + its own which Blogger also has for the most part; Blogger is limited to a .blog domain and slightly different from Personal.

**Comment by jaswrks** [comment]
Got it, thanks! We may want to update this tooltip then, changing it to "free"?

**Comment by jaswrks** [comment]
Also noticed this.

File: client/lib/plans/constants.js [hunk 5]
```diff
@@ -1097,7 +1108,7 @@ export const FEATURES_LIST = {
 	[ FEATURE_ALL_FREE_FEATURES ]: {
 		getSlug: () => FEATURE_ALL_FREE_FEATURES,
 		getTitle: () => i18n.translate( 'All free features' ),
-		getDescription: () => i18n.translate( 'Also includes all features offered in the Free plan.' ),
+		getDescription: () => i18n.translate( 'Also includes all features offered in the free plan.' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/lib/plans/constants.js [hunk 6]
```diff
@@ -1161,10 +1172,10 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_FREE_BLOG_DOMAIN ]: {
 		getSlug: () => FEATURE_FREE_BLOG_DOMAIN,
-		getTitle: () => i18n.translate( 'Free .blog Domain for One Year' ),
+		getTitle: () => i18n.translate( 'Free .blog domain for one year' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 18 translations.

**Comment by sararosso** [comment]
@jaswrks Great, thanks - fixed both of those.

File: client/lib/plans/constants.js [hunk 7]
```diff
@@ -291,13 +291,13 @@ const getPlanPremiumDetails = () => ( {
...
-	getAudience: () => i18n.translate( 'Best for entrepreneurs' ),
-	getBlogAudience: () => i18n.translate( 'Best for professionals' ),
+	getAudience: () => i18n.translate( 'Best for freelancers' ),
+	getBlogAudience: () => i18n.translate( 'Best for freelancers' ),
 	getPortfolioAudience: () => i18n.translate( 'Best for freelancers' ),
 	getStoreAudience: () => i18n.translate( 'Best for freelancers' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Entrepreneurs & Freelancers:{{/strong}}' +
+			'{{strong}}Best for Freelancers:{{/strong}}' +
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: Hi! I've found a possible matching string that has already been translated **31** times:
`translate( '{{strong}}Best for Entrepreneurs & Freelancers:{{/strong}} Build a unique website with advanced design tools, CSS editing, lots of space for audio and video, and the ability to monetize your site with ads.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by sararosso** [comment]
Going to break this PR into smaller ones to see if we can isolate what's breaking the CircleCI Tests.

**Pull request closed without merging**